packages/indexer-common/src/errors.ts

```diff
@@ -127,6 +128,7 @@ export const INDEXER_ERROR_MESSAGES: Record<IndexerErrorCode, string> = {
   IE054: 'Failed to collect receipts in exchange for query fee voucher',
   IE055: 'Failed to redeem query fee voucher',
   IE056: 'Failed to remember allocation for collecting receipts later',
+  IE057: 'Transaction reverted: contract requirement not met',
```

Comment:
What's a "contract requirement"? I feel this is confusing.

Reply:
It results from a revert() or require() in the contract.  I felt "contract requirement" sufficiently generally covered both of those cases.  

It'd be nice to include the actual revert reason string, but for now I'm open to suggestions for a general description.

Reply:
How about `Transaction reverted due to failing assertion in contract`? Essentially, that's what `revert()`, `require()` and `assert()` are: assertions.

Reply:
True, sounds good. 👍